rust/operator/src/kafka_controller.rs

```diff
@@ -55,16 +55,15 @@ use std::{
 };
 use strum::{EnumDiscriminants, IntoStaticStr};
 
-use crate::command::{get_svc_container_cmd_args, kcat_container_cmd_args};
 use crate::{
-    command,
+    command::{self, get_svc_container_cmd_args, kcat_container_cmd_args},
     discovery::{self, build_discovery_configmaps},
     pod_svc_controller,
-    utils::{self, ObjectRefExt},
+    utils::{self, build_recommended_labels, ObjectRefExt},
     ControllerConfig,
 };
 
-const RESOURCE_SCOPE: &str = "kafka-operator_kafkacluster";
+pub const KAFKA_CONTROLLER_NAME: &str = "kafkacluster";
```

Comment:
Isn't the controller name `kafka-controller` ?

Reply:
Its the controller responsible for the kafkacluster. What is confusing about this?